--- a/config.def.h	2025-12-15 22:28:01.589534258 -0500
+++ b/config.def.h	2025-12-15 22:12:30.736022112 -0500
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "DejaVu Sans Mono:style=Book:size=12";
 static int borderpx = 2;
 
 /*
@@ -96,35 +96,31 @@
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
+	"#282828", /* black (bg0) */
+	"#cc241d", /* red */
+	"#98971a", /* green */
+	"#d79921", /* yellow */
+	"#458588", /* blue */
+	"#b16286", /* magenta/purple */
+	"#689d6a", /* cyan/aqua */
+	"#a89984", /* white (gray) */
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
+	"#928374", /* bright black (gray) */
+	"#fb4934", /* bright red */
+	"#b8bb26", /* bright green */
+	"#fabd2f", /* bright yellow */
+	"#83a598", /* bright blue */
+	"#d3869b", /* bright magenta/purple */
+	"#8ec07c", /* bright cyan/aqua */
+	"#ebdbb2", /* bright white (fg0) */
 	[255] = 0,
-
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
-	"gray90", /* default foreground colour */
-	"black", /* default background colour */
+	"#ebdbb2", /* default foreground colour (fg) */
+	"#928374", /* cursor color */
+	"#ebdbb2", /* default foreground colour */
+	"#282828", /* default background colour (bg) */
 };
 
-
 /*
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
