diff --git a/someblocks.c b/someblocks.c
index f149896..af27589 100644
--- a/someblocks.c
+++ b/someblocks.c
@@ -34,7 +34,8 @@ void setupsignals();
 void sighandler(int signum);
 int getstatus(char *str, char *last);
 void statusloop();
-void termhandler();
+void termhandler(int);
+void sigpipehandler(int);
 void pstdout();
 void psomebar();
 static void (*writestatus) () = psomebar;
@@ -178,12 +179,12 @@ void sighandler(int signum)
 	writestatus();
 }
 
-void termhandler()
+void termhandler(int)
 {
 	statusContinue = 0;
 }
 
-void sigpipehandler()
+void sigpipehandler(int)
 {
 	close(somebarFd);
 	somebarFd = -1;
